test/Base.t.sol

```diff
@@ -33,6 +33,9 @@ abstract contract Base_Test is PRBTest, StdCheats, PRBMathAssertions, PRBMathUti
     /// @dev The maximum value an uint40 number can have.
     uint128 internal constant MAX_UINT40 = type(uint40).max;
 
+    /// @dev The minimum value an int256 number can have.
+    int256 internal constant MIN_INT256 = type(int256).min;
+
```

Comment:
Maybe should define it in `PRBTest` instead, where `MAX_INT256` is declared, so that MIN and MAX is declared at the same place

Reply:
Good idea! However, adding that constant in PRBTest would require a version bump. I will do this later, and proceed with merging the PR now.